src/Microsoft.Crank.PullRequestBot/Program.cs

```diff
@@ -170,6 +170,8 @@ private static async Task<int> Controller(BotOptions options)
 
             _configuration = await LoadConfigurationAsync(_options.Config);
 
+            CreateThumbprint();
```

Comment:
NIT: is it worth it to add a comment here explaining why we need to make sure this is called here for future reference?